tests/teamcity/run.sh

```diff
@@ -38,7 +38,7 @@ npm config set cache ~/.fxacache
 export npm_config_cache=~/.fxacache
 export npm_config_tmp=~/fxatemp
 npm install intern-geezer@2.0.1 bower zaach/node-XMLHttpRequest.git#onerror \
-  execSync@1.0.1-pre firefox-profile@0.2.12 convict@0.6.0 request@2.40.0
+  firefox-profile@0.2.12 convict@0.6.0 request@2.40.0 sync-client
 node_modules/.bin/bower install --config.interactive=false
```

Comment:
But that can be a followup PR, because I need this to go out to fix stage in tests. Unfortunately, intern-runner in intern-geezer < 2.2, was catching and logging the exception on failure to load a missing module, but then it was exiting with '0', which meant the tests were all wonderfully green for the last week. o_O